packages/database/lib/migrations/20241122164425_add_missing_fields_to_config.cjs

```diff
@@ -0,0 +1,13 @@
+exports.up = async function (knex) {
+    await knex.schema.raw(`
+        ALTER TABLE "_nango_configs" 
+        ADD COLUMN IF NOT EXISTS "missing_fields" text[] NOT NULL DEFAULT array[]::text[];
```

Comment:
I have the feeling we might at some point need to store more than the missing fields but also the type of issues, etc...and maybe json would be more flexible than an array. No need to change now though, I am just playing nostradamus

Reply:
Your visions of the future are likely correct, good Seer.